src/common/libidset/idset.h

```diff
@@ -40,6 +40,10 @@ enum idset_flags {
 struct idset *idset_create (size_t size, int flags);
 void idset_destroy (struct idset *idset);
 
+/* Get the current number of slots in the idset.
```

Comment:
Because I'm not certain what that means either is that just the total number of inegers that can currently be set in the idset? How is this related to what the rest of the documentation calls 'size'?

Reply:
Yes, this.

We should use consistent terminology so I'll see about tighting that up.  Just _size_ by itself is amgiguous.  For example, one might interpret it as the universe size (the number of possible IDs), the set size (number of IDs in the set), or the number of bytes consumed by the data structure.